packages/backend/src/controllers/exploreController.ts

```diff
@@ -143,7 +143,7 @@ export class ExploreController extends BaseController {
             columnOrder: string[];
             hiddenFields?: string[];
             chartName?: string;
-            pivotColumns?: string[];
+            pivotConfig?: PivotConfig;
```

Comment:
I'm refactoring the pivotConfig api, to use pivotCOnfig instead of separate parameters for pivotColumns and metricsAsRows.